sources/packages/api/src/cra-integration/cra-integration.models.ts

```diff
@@ -76,6 +76,68 @@ export enum MatchStatusCodes {
   unsuccessfulMatch = "50",
 }
 
+/**
+ * SIN match status codes (SIN-TTN-STATUS-CODE) presents on
+ * CRA Response Record (Trans Sub Code - 0022).
+ */
+export enum SINMatchStatusCodes {
```

Comment:
The enum must start with an upper case as per the wiki: https://github.com/bcgov/SIMS/wiki/Development-Standards-and-Decisions#how-to-declare-an-enum


![image](https://user-images.githubusercontent.com/61259237/156837755-d94a6280-7693-40c5-bd14-c08a1132bbce.png)